apps/choleskey/generateInputMatrix.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This is an Jupyter Script for generating the Input Positive Definition(PD) Matrix for Cholesky decomposition.
+# The generated .txt file stores the entire PD matrix with the matrix elems row by row. 
+# This input.txt file can be used directly In benchmark part in each cpp file.
+
+
+import numpy as np
+
+#enter matrix size
+matrixSize = 4
```

Comment:
Hardcoding this is not a good idea. Consider using `argparse` module and use that to take input from the user on the matrix size to generate.